x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/deployment/DeploymentManager.java

```diff
@@ -588,8 +607,15 @@ private boolean isThisProcessOlderThan1Day() {
             return startTime.isBefore(Instant.now().minus(Duration.ofDays(1)));
         }
 
-        private void finishClosingProcess(AtomicInteger startsCount, String reason) {
-            logger.warn("[{}] inference process failed after [{}] starts, not restarting again", task.getDeploymentId(), startsCount.get());
+        private void finishClosingProcess(AtomicInteger startsCount, String reason, String deploymentId) {
```

Comment:
Just curious does the `reason` get logged later?

Reply:
I'm not sure what you mean. It does get logged just below here.